### DISCUSSION TITLE
Backport procedure?

### DISCUSSION THREAD

**Discussion by lolbinarycat** [fish-shell/fish-shell#10872] [Q&A] [answered]
I've been struggling with https://github.com/fish-shell/fish-shell/issues/10379 for some time now, but I'm on NixOS so building from source is prohibitively difficult.

I would be happy to help with backporting a fix to a potential 3.7.2, but there doesn't seem to be any procedure for doing so, and there's no news for when we'll be getting another release.

**Comment by faho** [comment, marked as answer]
> I would be happy to help with backporting a fix to a potential 3.7.2

There will likely not be a 3.7.2.

> there doesn't seem to be any procedure for doing so

We don't really tend to do multiple release tracks. We're about five people, so that's a lot of our effort invested that could also go into new stuff.

-----

> building from source is prohibitively difficult

For completions like that, you don't have to build from source. You can just download https://github.com/fish-shell/fish-shell/blob/0ff1e50a33cc494ee2cd5f1490008e3e3e94e5df/share/completions/rustc.fish and stick it in ~/.config/fish/completions/. Or the same file on master: https://github.com/fish-shell/fish-shell/blob/master/share/completions/rustc.fish

Note that this isn't guaranteed to work - we do sometimes use new options in our completions - but in this case I think it should.
